### ISSUE TITLE
Spotlight paper with no annotation

### ISSUE THREAD

**Issue by kimrutherford** [pombase/curation#3720]
Just in case this was an oversight:

https://www.pombase.org/reference/PMID:38442865

"This publication has no annotations in PomBase because it hasn't been curated yet. "

**Comment by ValWood** [comment]
Yep we need to decide on GO terms but it turned out more complicated. 
Will sort.

**Comment by ValWood** [comment]
the session is through but pending  new GO terms
